packages/datadog-plugin-kafkajs/test/index.spec.js

```diff
@@ -0,0 +1,139 @@
+'use strict'
+
+const { expect } = require('chai')
+const agent = require('../../dd-trace/test/plugins/agent')
+const { expectSomeSpan, withDefaults } = require('../../dd-trace/test/plugins/helpers')
+const plugin = require('../src')
+
+wrapIt()
+
+const TIMEOUT = 30000
```

Comment:
This shouldn't need to be changed and there should instead be a `services/kafka.js` instead that handle the wait and any required setup before the test starts.